src/views/SafetyStock.vue

```diff
@@ -70,6 +73,7 @@ async function fetchRules() {
   store.dispatch("util/updateSelectedSegment", "");
   store.dispatch("rule/updateIsReorderActive", false)
   await Promise.allSettled([store.dispatch('rule/fetchRules', { groupTypeEnumId: 'RG_SAFETY_STOCK', pageSize: 50 }), store.dispatch("util/fetchConfigFacilities"), store.dispatch("util/fetchFacilityGroups")]);
+  await store.dispatch('rule/fetchArchivedRules')
```

Comment:
We should think on fetching archived rules with active rules and then separate them based on status, let me know if we have any issues doing this.